app/assets/javascripts/host_edit.js

```diff
@@ -108,7 +108,6 @@ function computeResourceSelected(item) {
         if ($('#compute_resource').find('.alert-danger').length > 0)
           $('#compute_resource_tab a').addClass('tab-error');
         update_capabilities($('#capabilities').val());
-        tfm.numFields.initAll();
```

Comment:
This would also be still needed by plugins using the old components

Reply:
I believe we now don't have core components depending on it, so I'd say we can get rid of that, in case plugins are using the spinners directly with their own inputs, they are responsible for the components initialization and we should not keep dead code for that here.
**Note** I'm writing this in a strong believe that plugins are initializing this by themselves and do not relly on this to initialize for them.